script.js

```diff
@@ -0,0 +1,75 @@
+function Book(title, author) {
+  this.title = title;
+  this.author = author;
+}
+
+let books = [];
+
+/* eslint-disable no-use-before-define */
+function removeBook(index) {
+  books.splice(index, 1);
+  storeBookLocally(books);
+  displayItems();
+}
```

Comment:
- [OPTIONAL] I highly recommend you use the `filter` method which does not mutate the current array instead of using the `splice` method.